addons/godot_pixelorama_importer/spriteframes_import.gd

```diff
@@ -81,10 +81,12 @@ func import(source_file, save_path, options, _r_platform_variants, r_gen_files):
 		frames.set_animation_speed(tag.name, options.animation_fps)
 
 		for frame in range(tag.from, tag.to + 1):
-			var atlas = AtlasTexture.new()
-			atlas.atlas = spritesheet_tex
-			atlas.region = Rect2(Vector2((frame - 1) * frame_size.x, 0), frame_size)
-			frames.add_frame(tag.name, atlas)
+			var image_rect := Rect2(Vector2((frame - 1) * frame_size.x, 0), frame_size)
+			var image := Image.new()
+			image = spritesheet_tex.get_data().get_rect(image_rect)
```

Comment:
This feels redundant, was it necessary to keep it like this?

Reply:
I tried changing it to a single line `var image = spritesheet_tex.get_data().get_rect(image_rect)` but it seems to be causing the same issue that AtlasTexture does. I am not sure why this happens, but it seems to require a new Image resource in order for it to work.

Reply:
That's real odd, hmm :thinking: